lib/components/MarkdownDisplayer/MarkdownDisplayer.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react'
+import { useTheme } from '@apisuite/fe-base'
+import Editor from 'rich-markdown-editor'
+
+import { MarkdownDisplayerProps } from './types'
+
+export const MarkdownDisplayer: React.FC<MarkdownDisplayerProps> = ({
+  content,
+}) => {
+  const { palette, typography } = useTheme()
+
+  /* For us to 'brand' our markdown, we need to completely override
+  the markdown displayer component's (i.e., 'Editor') in-built theme
+  (even if we only mean to override some of its properties).
+  
+  Please refer to 'https://github.com/outline/rich-markdown-editor' for more details. */
+  const markdownDisplayerTheming = {
```

Comment:
Import theme and overwrite its properties.

```
import Editor, { theme } from 'rich-markdown-editor'

const customTheme = {
    ...theme,
    code: palette.secondary.contrastText,
   ...
 }
```

Reply:
Good suggestion, but keep in mind that some properties will forcibly be present and set to null because their default does not provide sufficient contrast.

Reply:
Example (for future reference):

<img width="491" alt="Screenshot 2021-07-21 at 11 41 37" src="https://user-images.githubusercontent.com/67902809/126476139-f041cb4f-9b19-4a49-b858-c6b830a4114c.png">